### PR TITLE
Remove Character Restriction in Bounty Editing

### PR DESCRIPTION
**Pull request by saithsab877**
### Problem:
When attempting to edit a bounty, users encounter a character limit restriction. This issue likely stems from an inconsistency in handling character limits between the "edit bounty" and "edit organization" modals. This restriction impedes users' ability to fully describe or update the bounty details, potentially affecting the clarity and effectiveness of the bounty's description.

### Expected Behavior:
The expected behavior is for users to have the ability to edit a bounty without facing a character limit. 

## Issue ticket number and link:
- **Ticket Number:** [ 156 ]
- **Link:** [ https://github.com/stakwork/sphinx-tribes-frontend/issues/156 ]

### Evidence:
 Please see the attached video as evidence.
- Demo: [Link ](https://www.loom.com/share/76496067bf3d415895b629884ffcc700)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by saithsab877** [comment]
Hi @ecurrencyhodler, @aliraza556,
Please review this PR. Thank you.

**Comment by aliraza556** [comment]
Hi @saithsab877,
Thank you for submitting your first bounty. Let's review your code and functionality.

**Comment by aliraza556** [comment]
I examined the issue and the related code modifications and carried out local testing to verify that everything functions correctly without impacting other features.
Hi @saithsab877,
Thank you for checking every edge case.

**Comment by saithsab877** [comment]
Hi @kevkevinpal,
Please review this PR.

File: src/components/form/inputs/TextAreaInputNew.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ export default function TextAreaInputNew({
         }}
         onChange={(e: any) => {
           const newVal = e.target.value;
-          if (name === 'description') {
+          if (name === 'org_description') {
```
Comment (elraphty) [inline src/components/form/inputs/TextAreaInputNew.tsx:107]: @saithsab877 this breaks the add org description value if the description is changed to org_description Formik won't be able to pick up the value of the description on submission

```sh

      const newOrg = {
        id: org.id,
        uuid: org.uuid,
        name: body.name || org.name,
        owner_pubkey: org.owner_pubkey,
        img: img || org.img,
        description: body.description || org?.description,
        github: body.github || org?.github,
        website: body.website || org?.website,
        created: org.created,
        updated: org.updated,
        show: body?.show !== undefined ? body.show : org.show,
        bounty_count: org.bounty_count,
        budget: org.budget
      };
```

Comment (saithsab877) [reply src/components/form/inputs/TextAreaInputNew.tsx:107]: Hi @elraphty,
Got it, thanks for the heads up! I'll change both the field name to 'org_description' and the reference in the body accordingly to keep everything in sync for Formik.

**Comment by saithsab877** [comment]
Hi @elraphty,

I've implemented the necessary changes for the description field name to `org_description` as discussed. This includes updating both the field name and all references in the body to ensure Formik correctly processes the form submission. Please review the PR at your earliest convenience to ensure everything aligns with our expectations. I appreciate your guidance on this and look forward to your feedback.

Thank you!

**Pull request closed without merging**